api/src/main/java/org/openmrs/module/ugandaemrreports/reporting/calculation/smc/AnaesthesiaCalculation.java

```diff
@@ -0,0 +1,60 @@
+/**
+ * The contents of this file are subject to the OpenMRS Public License
+ * Version 1.0 (the "License"); you may not use this file except in
+ * compliance with the License. You may obtain a copy of the License at
+ * http://license.openmrs.org
+ *
+ * Software distributed under the License is distributed on an "AS IS"
+ * basis, WITHOUT WARRANTY OF ANY KIND, either express or implied. See the
+ * License for the specific language governing rights and limitations
+ * under the License.
+ *
+ * Copyright (C) OpenMRS, LLC.  All Rights Reserved.
+ */
+package org.openmrs.module.ugandaemrreports.reporting.calculation.smc;
+
+import org.openmrs.Obs;
+import org.openmrs.calculation.patient.PatientCalculationContext;
+import org.openmrs.calculation.result.CalculationResultMap;
+import org.openmrs.calculation.result.SimpleResult;
+import org.openmrs.module.ugandaemrreports.reporting.calculation.AbstractPatientCalculation;
+import org.openmrs.module.ugandaemrreports.reporting.calculation.Calculations;
+import org.openmrs.module.ugandaemrreports.reporting.calculation.EmrCalculationUtils;
+import org.openmrs.module.ugandaemrreports.reporting.metadata.Dictionary;
+
+import java.util.Collection;
+import java.util.Map;
+
+/**
+ * Created by Nicholas Ingosi on 7/17/17.
+ */
+public class AnaesthesiaCalculation extends AbstractPatientCalculation{
```

Comment:
That calculation is only called once in the data definition **Type anasthesia column**